scripts/run-integration-tests.sh

```diff
@@ -7,23 +7,84 @@ cd "${PROJECT_ROOT}" || exit 1
 
 source "scripts/testlib.sh"
 check-set-swap-test-data-dir
-mkdir -p "${SWAP_TEST_DATA_DIR}/"{alice,bob,charlie}
 
-# Charlie uses deterministic ganache key #49
-CHARLIE_ETH_KEY="${SWAP_TEST_DATA_DIR}/charlie/eth.key"
-echo "87c546d6cb8ec705bea47e2ab40f42a768b1e5900686b0cecc68c0e8b74cd789" >"${CHARLIE_ETH_KEY}"
+RELAYER_PORT=7799
+
+# First 5 deterministic ganache keys (only 4 used at the moment)
```

Comment:
```suggestion
# First 5 deterministic ganache keys (only 4 using at the moment)
```

Reply:
also we can just have 4 keys here for now, if only 4 are used?

Reply:
I over-reserved 5 keys in `tests/ganache.go` so that there would be a spare key for future use that wouldn't require modifying anything other than this script. Do you want me to change it in both places?